Core/src-test/com/infiniteautomation/mango/spring/service/MailingListServiceTest.java

```diff
@@ -3,6 +3,8 @@
  */
 package com.infiniteautomation.mango.spring.service;
 
+import static org.junit.Assert.*;
```

Comment:
We shouldn't be adding wildcard imports.  This should be configured in your IDE. Please check the setup doc and fix.

Reply:
I missed the static import setting, thanks.